controllers/rhmi/rhmi_controller.go

```diff
@@ -1120,7 +1116,7 @@ func (r *RHMIReconciler) preflightChecks(installation *rhmiv1alpha1.RHMI, instal
 	if err != nil {
 		log.Infof("error updating status", l.Fields{"error": err.Error()})
 	}
-	return result, nil
+	return nil
```

Comment:
same as above

Reply:
Generally, anywhere where we return result previously, we need to return non-nil value to ensure the reconciler gets re-queued unless you are 100% that we do not need to re-queue but instead, can continue within same reconcile loop.